components/brave_shields/common/BUILD.gn

```diff
@@ -14,12 +14,20 @@ static_library("common") {
   deps = [
     "//base",
     "//components/content_settings/core/common",
+    "//brave/components/resources:strings",
     "//url",
+    "//ui/base",
   ]
 }
 
 mojom("mojom") {
-  sources = [ "brave_shields.mojom" ]
+  sources = [
+    "brave_shields.mojom",
+    "brave_shields_panel.mojom",
+  ]
 
-  deps = [ "//mojo/public/mojom/base" ]
+  public_deps = [
+    "//mojo/public/mojom/base",
+    "//url/mojom:url_mojom_gurl",
```

Comment:
Using `public_deps` here seems good but why did you add `url_mojom_gurl`?
It seems shields panel doesn't use this

Reply:
my fault, will remove